Packs/DefaultPlaybook/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### New: Default
+- This playbook executes when no other playbook is associated with an incident. It enriches indicators in an incident using one or more integrations. (Available from Cortex XSOAR 8.0.0).
```

Comment:
@idovandijk let's edit release notes as 8.0.0 is not out yet. 

Please add the following comment instead:
<!-- future version -->